packages/ckeditor5-basic-styles/src/italic.ts

```diff
@@ -26,14 +28,20 @@ export default class Italic extends Plugin {
 	/**
 	 * @inheritDoc
 	 */
-	static get requires() {
+	public static get requires(): PluginDependencies {
 		return [ ItalicEditing, ItalicUI ];
 	}
 
 	/**
 	 * @inheritDoc
 	 */
-	static get pluginName() {
+	public static get pluginName(): 'Italic' {
 		return 'Italic';
 	}
 }
+
+declare module '@ckeditor/ckeditor5-core' {
+	interface CommandsMap {
+		italic: AttributeCommand;
+	}
+}
```

Comment:
Please move this and other similar declarations to the appropriate editing files.